src/VisualStudio/Core/Def/DesignerAttribute/VisualStudioDesignerAttributeService.cs

```diff
@@ -5,24 +5,24 @@
 using System;
```

Comment:
the meat of the change is in this type.  This is the host-side of the feature.  so this is the side that 'pulls' on the remote side to do the computation, then 'pushes' that data into the project system.

@DustinCampbell @tmeschter, ideally in teh future we can just have teh project system pull on this data using LSP, instead of this hybrid push/pull model.